UKHO.SalesCatalogueFileShareServicesMock.API/UKHO.SalesCatalogueFileShareServicesMock.API/Services/FileShareService.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Microsoft.Extensions.Options;
+using System;
+using System.IO;
+using System.Linq;
+using UKHO.SalesCatalogueFileShareServicesMock.API.Common;
+using UKHO.SalesCatalogueFileShareServicesMock.API.Helpers;
+using UKHO.SalesCatalogueFileShareServicesMock.API.Models.Response;
+
+namespace UKHO.SalesCatalogueFileShareServicesMock.API.Services
+{
+    public class FileShareService
+    {
+        private readonly IOptions<FileShareServiceConfiguration> fileShareServiceConfiguration;
+
+        public FileShareService(IOptions<FileShareServiceConfiguration> fileShareServiceConfiguration)
+        {
+            this.fileShareServiceConfiguration = fileShareServiceConfiguration;
+        }
+
+        public SearchBatchResponse GetBatches(string filter)
+        {
+            var response = FileHelper.ReadJsonFile<SearchBatchResponse>(fileShareServiceConfiguration.Value.FileDirectoryPath + fileShareServiceConfiguration.Value.ScsResponseFile);
+            if (filter.Contains("README.TXT", StringComparison.OrdinalIgnoreCase))
+            {
+                response.Entries.RemoveRange(1, response.Entries.Count - 1);
+            }
+            return response;
+        }
+
+        public byte[] GetFileData(string filesName)
+        {
+            string filePath, fileType = Path.GetExtension(filesName);
```

Comment:
better to declare `filePath` where it is used

Reply:
Yes. Did changes accordingly. Thanks!